internal/bminventory/inventory.go

```diff
@@ -820,6 +820,7 @@ func (b *bareMetalInventory) updateImageInfoPostUpload(ctx context.Context, infr
 				return errors.New("Failed to generate image: error generating URL")
 			}
 		} else {
+			// TODO(djzager): Needs to be updated with image-service work, MGMT-3934
```

Comment:
Do you want to commit this?

Reply:
I think I'll remove this. Will wait to see if I have any issues verifying this manually.

Reply:
Decided to leave it so I can find it when it comes time to address MGMT-3934